src/patternfly/components/Page/page.scss

```diff
@@ -63,12 +63,24 @@
   --pf-c-page__header-tools--MarginBottom: var(--pf-global--spacer--sm);
   --pf-c-page__header-tools--md--MarginRight: var(--pf-global--spacer--lg);
   --pf-c-page__header-tools--c-avatar--MarginLeft: var(--pf-global--spacer--md);
-  --pf-c-page__header-tools-group--MarginLeft: var(--pf-global--spacer--xl);
+  --pf-c-page__header-tools-group--Display: flex;
+  --pf-c-page__header-tools-group--m-icons--c-button--Display: block;
+  --pf-c-page__header-tools-group--m-icons--c-dropdown--Display: block;
+  --pf-c-page__header-tools-group--c-avatar--Display: inline-block;
```

Comment:
What do you think about adding a `.pf-c-page__header-tools-item` that can be a child of `.pf-c-page__header-tools-group`, then we apply the hidden-visible extend to the `tools-item` and `tools-group` classes? Then a user can put whatever they want in that and we won't need to add hidden-visible support for each thing we/users add to the header tools (buttons and dropdowns currently).

So each button/dropdown in a group would be wrapped in an item, and the avatar would be wrapped in an item. That's also a more modular approach.

Reply:
Ohh I like that idea. I updated the PR